test/integration/run-test-v2.sh

```diff
@@ -66,7 +66,15 @@ helm install azuredisk-csi-driver test/latest/azuredisk-csi-driver -n kube-syste
   --set image.azuredisk.tag=$image \
   --set azuredisk.cloudConfig="$(cat "${AZURE_CREDENTIAL_FILE}" | base64 | awk '{printf $0}'; echo)" \
   --set controller.port="${CONTROLLER_PORT}" \
+  --set azuredisk.nodeId="${nodeid_0}"
```

Comment:
Unfortunately, this line accidentally omitted the trailing backslash which results in the Azure credential file contents being written to the log when Helm dumps the chart output. I just now submitted a fix for this to the Mandalay master branch. Please pull that change into this PR.